src/webgui/src/webgui/image_pub.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+import sys
+import rospy
+import cv2
+from std_msgs.msg import String
+from sensor_msgs.msg import Image
+from cv_bridge import CvBridge, CvBridgeError
+import rospy
+import time
+
+
+class image_pub:
+	""" Class for publishing images from video source
+
+    Attributes:
```

Comment:
It looks like the indentation is ever so slightly off, though that might just be how github is showing it